docusaurus/docs/introduction/plugin-types-usage.md

```diff
@@ -77,13 +77,13 @@ Not all plugins support this type of configuration. Refer to the readme of speci
 
 ### Provisioning of data sources
 
-Data sources can also be provisioned using Grafana’s [provisioning features](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources) so that you can [efine Grafana as-code as part of a GitOps approach when Grafana starts up or [on demand](https://grafana.com/docs/grafana/latest/developers/http_api/admin/#reload-provisioning-configurations). 
+Data sources can also be provisioned using Grafana’s [provisioning features](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources) so that you can define Grafana as-code as part of a GitOps approach when Grafana starts up or [on demand](https://grafana.com/docs/grafana/latest/developers/http_api/admin/#reload-provisioning-configurations). 
 
 Note that a plugin of type `datasource` must be installed before it can be provisioned.
 
 ### Bundling of dashboards
 
-Data-source plugins can [include dashboards]((https://grafana.com/developers/plugin-tools/reference-plugin-json#includes)) by referencing dashboard JSON files (including `property` and `type=dashboard`) within the `plugin.json` file. Grafana puts a dashboard in the `General` folder when it is imported.
+Data-source plugins can [include dashboards](https://grafana.com/developers/plugin-tools/reference-plugin-json#includes) by referencing dashboard JSON files (including `property` and `type=dashboard`) within the `plugin.json` file. Grafana puts a dashboard in the `General` folder when it is imported.
```

Comment:
We shouldn't navigate from plugin-tools docs to the live site.  I believe we can shorten these to `/<path-to-page>`.

```suggestion
Data-source plugins can [include dashboards](/reference-plugin-json#includes) by referencing dashboard JSON files (including `property` and `type=dashboard`) within the `plugin.json` file. Grafana puts a dashboard in the `General` folder when it is imported.
```